backend/app/Savor22b.Tests/Action/PlaceUserHouseActionTests.cs

```diff
@@ -1,64 +1,187 @@
 namespace Savor22b.Tests.Action;
 
 using System;
-using Libplanet.Crypto;
 using Libplanet.State;
 using Xunit;
 using Savor22b.Action;
 using Savor22b.States;
-using Libplanet;
 using Savor22b.Constants;
+using Libplanet.Assets;
+using Savor22b.Action.Exceptions;
 
 public class PlaceUserHouseActionTests : ActionTests
 {
-    public PlaceUserHouseActionTests()
-    {
-    }
+    public PlaceUserHouseActionTests() { }
 
-    [Theory]
-    [InlineData(1, 0, 0)]
-    [InlineData(1, 1, 1)]
-    [InlineData(2, 0, 0)]
-    [InlineData(2, 1, 1)]
-    [InlineData(2, 2, 6)]
-    public void PlaceUserHouseActionExecute_UpdateUserHouseStateToExistsUserHouseState(
-        int villageId,
-        int targetX,
-        int targetY
-    )
+    [Fact]
+    public void Execute_Success_InitialPlaceHouse()
     {
-        IAccountStateDelta state = new DummyState();
-        RootState rootState = new RootState();
+        int villageId = 1;
+        int targetX = 0;
+        int targetY = 0;
 
-        var random = new DummyRandom(1);
+        IAccountStateDelta state = new DummyState();
 
         var action = new PlaceUserHouseAction(villageId, targetX, targetY);
 
-        state = action.Execute(new DummyActionContext
-        {
-            PreviousStates = state,
-            Signer = SignerAddress(),
-            Random = random,
-            Rehearsal = false,
-            BlockIndex = 1,
-        });
+        state = action.Execute(
+            new DummyActionContext
+            {
+                PreviousStates = state,
+                Signer = SignerAddress(),
+                Random = random,
+                Rehearsal = false,
+                BlockIndex = 1,
+            }
+        );
 
         var rootStateEncoded = state.GetState(SignerAddress());
-        rootState = rootStateEncoded is Bencodex.Types.Dictionary bdict
+        RootState rootState = rootStateEncoded is Bencodex.Types.Dictionary bdict
             ? new RootState(bdict)
             : throw new Exception();
 
+        var globalUserHouseStateEncoded = state.GetState(Addresses.UserHouseDataAddress);
+
+        GlobalUserHouseState globalUserHouseState = globalUserHouseStateEncoded
+            is Bencodex.Types.Dictionary stateEncoded
+            ? new GlobalUserHouseState(stateEncoded)
+            : throw new Exception();
+
         Assert.Equal(villageId, rootState.VillageState!.HouseState.VillageID);
         Assert.Equal(targetX, rootState.VillageState.HouseState.PositionX);
         Assert.Equal(targetY, rootState.VillageState.HouseState.PositionY);
+        Assert.Equal(
+            SignerAddress(),
+            globalUserHouseState.UserHouse[$"{villageId},{targetX},{targetY}"]
+        );
+    }
+
+    private IAccountStateDelta createStateForReplaceHouse()
```

Comment:
```suggestion
    private IAccountStateDelta createStateForRelocationHouse()
```

다른곳에 다 relocation으로 되어있는 것 같아서요...

Reply:
헉 제가 정신 놓고 썼나봐요 감사합니다람쥐